### ISSUE TITLE
Potential conflict with LiteSpeed cache

### ISSUE THREAD

**Issue by over-engineer** [pressidium/pressidium-cookie-consent#45]
Initially reported by @formaciongrafica in #43.

>When I clear the LiteSpeed cache, while editing the Pressidium options, on several occasions the Pressidium plugin is misconfigured and does not correct itself even if I force it to update.

>The possible bug leads to all previous plugin configurations being lost.

There might be some sort of conflict with the [LightSpeed Cache](https://wordpress.org/plugins/litespeed-cache/) plugin.

**Comment by over-engineer** [comment]
Could not reproduce in a vanilla WordPress installation with the following plugins installed:

- [Pressidium Cookie Consent](https://wordpress.org/plugins/pressidium-cookie-consent/) at version `1.2.2` (this plugin)
- [LightSpeed Cache](https://wordpress.org/plugins/litespeed-cache/) at version `5.7.0.1`

![lightspeed_cache_purge_all](https://github.com/pressidium/pressidium-cookie-consent/assets/6157348/c1865bca-cf1b-433b-8cf2-edc90c58f943)

Purging all caches does not seem to be causing any issues with the Pressidium Cookie Consent plugin.

**Comment by over-engineer** [comment]
@formaciongrafica I tried to replicate the issue on my end, but, unfortunately, I couldn’t reproduce it 😔

Here’s a few things you could check:

- Update plugins to their latest versions to address any known issues

- If possible, temporarily deactivate all WordPress plugins _except_ **Pressidium Cookie Consent** and **LiteSpeed Cache** to see if the issue persists. Re-enable them one by one to identify any conflicting plugins

- Inspect the console, are there any errors? (on Google Chrome, CTRL + Shift + J on Windows, or CMD + Option + J on macOS)

- Check the error logs on your server

**Comment by formaciongrafica** [comment]
Hello Konstantinos
In reality I have been able to replicate the error in 3 different installations, and in all of them the same error is generated. I have disabled both plugins and the problem persists. The only factors in common are:
1. LiteSpeed + Pressidium plugins
2. Divi as a page builder.

I hope I have provided some more clues.
Thanks again!
Placid.

**Comment by formaciongrafica** [comment]
Hello Konstantinos
Sorry I couldn't answer you before.

I attach the screenshot of my Chrome error console, as well as the November and December logs of my hosting service with all the logs related to Pressidium and LiteSpeed. Regarding the .php.error.log there was no record related to either plugin.

I tried to reproduce the error with the new version of Pressidium installed and I couldn't replicate it again, so I assume it was some conflict with a previous plugin that I updated, or some change in the new version of Pressidium :)

Regarding the configuration of the Pressidium, I reconfigure it and send you the record, at the latest, during this afternoon or evening.

Thank you very much, again, for your constant assistance.
Placido.


[formaciongrafica.net-ssl_log.zip](https://github.com/pressidium/pressidium-cookie-consent/files/13538121/formaciongrafica.net-ssl_log.zip)
